src/__test__/Rockets.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import RocketsList from '../components/Rockets';
+import store from '../redux/configureStore';
+
+it('rockets list renders properly', () => {
+  const tree = renderer
+    .create(<Provider store={store}><RocketsList /></Provider>)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] **You have done a great job so far** However, while running the testing, there is an error ```src/__test__/Rockets.test.js``` that needs to be fixed to complete this project, 😎😎
    -  The error messages indicate that the component **RocketsList** is expecting some props that are not being passed in. The specific props referred to are title, description, image, and id. The error message also mentions that these props are marked as required in the RocketsList component, meaning their values cannot be undefined. 

     ![image](https://user-images.githubusercontent.com/102757126/217825860-2235b9ca-efe7-4acb-8ad4-e3a093c24ee2.png)
 
### KEEP IT UP 👍 
_Remember, tests ensure that your code works as expected, even when you make changes to it. This helps prevent bugs and reduces the likelihood of regressions._

Reply:
![Screenshot from 2023-02-09 15-20-09](https://user-images.githubusercontent.com/110976050/217838841-76df41ef-cfaf-466b-9fff-1e7661d91cc5.png)